### DISCUSSION TITLE
No Close button for manager?

### DISCUSSION THREAD

**Discussion by ShamanicArts** [Comfy-Org/ComfyUI-Manager#154] [Q&A] [answered]
Just installed comfyUI Manager on windows using second method , it seems to install fine but I have no close button? 

![image](https://github.com/ltdrdata/ComfyUI-Manager/assets/90007349/e349fb86-a13b-4994-80d1-9d13aa083a21)

Am I being an idiot (probably) or is something wrong? whenever ive messed with the manager on cloud installations I had a close button

**Comment by ltdrdata** [comment]
This is an issue occurring in a low-resolution environment. As a temporary workaround, zooming out via Ctrl-drag on the dialog resolves it, but there are plans to rework the layout in the current Manager due to the increasing number of controls.

**Comment by ltdrdata** [comment, marked as answer]
This is fixed since v0.37.
